pkg/packer/manifests/vsphere/base/packer-base.json

```
@@ -13,9 +13,9 @@
         "username": "{{env `VSPHERE_USERNAME`}}",
         "password": "{{env `VSPHERE_PASSWORD`}}",
         "vcenter_server": "{{env `VSPHERE_SERVER`}}",
-        "vsphere_guest_os_type": null,
-        "ssh_username": "",
-        "ssh_password": ""
+        "ssh_username": "{{env `SSH_USERNAME`}}",
```

```suggestion
        "ssh_password": null,
        "ssh_username": "{{env `SSH_USERNAME`}}",
```

IMO this could stay here as long as we do not set a default. There might be circumstances where users want to use it. So what about